model_zoo/vlm/api_model/model_adapter.py

```diff
@@ -124,28 +151,59 @@ def process_single_item(self, i, inter_results_dir):
                 data = json.load(f)
                 reason = data.get("reason", "")
                 result = data.get("answer", "")
-            return {
-                "question_id": question_id,
-                "question": qs,
-                "answer": result,
-                "reason": reason,
-            }
+                multiple_answers = data.get("multiple_answers", [])
+                return {
```

Comment:
We need to refactor the output in a subsequent PR, it's too complex. I recommend using a dataclass.